### PR TITLE
Updates to episodes 02 and 07

### PR DESCRIPTION
**Pull request by samhalsey**
This is a PR related to the issue for Keeping the sentences about config email address together in the lesson #487 and  Demonstrate drag n' drop file upload (new feature) #240

### CODE REVIEW

File: _episodes/07-github.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
 ---
-title: Remotes in GitHub
+title: "Remotes in GitHub"
+keypoints:
```
Comment (ErinBecker) [inline _episodes/07-github.md]: I think keypoints, questions, and objectives each need to be in quotes in order to render properly.

**Comment by kekoziar** [comment]
@samhalsey, thank you for this contribution.  There are a couple of requested changes, that I'm not sure you were notified of.  Will you reorder the metadata fields at the top of episode 7 to the following? 
title: 
teaching:
exercises: 
questions:
objectives:
keypoints:
Also, the questions, objectives, and keypoints need to be surrounded by double-quotes. While they are rendering on my machine, I suspect that may change at a whim of technology if the syntax isn't correct.  Thanks!

**Comment by munkm** [comment]
Hi @samhalsey, would you like some help from a maintainer on this PR?

File: _episodes/07-github.md (hunk 1 above)
Comment (kekoziar) [reply _episodes/07-github.md]: This is now fixed.

Comment (kekoziar) [reply _episodes/07-github.md]: This is now fixed.

**Pull request merged**